src/Reflection.php

```diff
@@ -19,19 +19,19 @@ class Reflection
     /**
      * Perform class reflection to create dispatch signatures
      *
-     * Creates a {@link \Laminas\Server\Reflection\ClassReflection} object for the class or
+     * Creates a {@link ReflectionClass} object for the class or
      * object provided.
      *
      * If extra arguments should be passed to dispatchable methods, these may
      * be provided as an array to $argv.
      *
      * @param string|object $class Class name or object
-     * @param  bool|array $argv Optional arguments to be used during the method call
+     * @param false|array $argv Optional arguments to be used during the method call
      * @param string $namespace Optional namespace with which to prefix the
-     * method name (used for the signature key). Primarily to avoid collisions,
-     * also for XmlRpc namespacing
-     * @return \Laminas\Server\Reflection\ReflectionClass
-     * @throws \Laminas\Server\Reflection\Exception\InvalidArgumentException
+     *                          method name (used for the signature key). Primarily to avoid collisions,
```

Comment:
Is this alignment correct?

Reply:
_I think_ ultimately we want to align all tags, variable names and descriptions.
I've added the indent here, just to increase readability. I can revert it to just 4 spaces indent.